src/NzbDrone.Api/ProviderModuleBase.cs

```diff
@@ -119,8 +119,7 @@ protected virtual void MapToResource(TProviderResource resource, TProviderDefini
 
             resource.Fields = SchemaBuilder.ToSchema(definition.Settings);
 
-            resource.InfoLink = string.Format("https://github.com/Radarr/Radarr/wiki/Supported-{0}#{1}",
-                typeof(TProviderResource).Name.Replace("Resource", "s"),
+            resource.InfoLink = string.Format("https://wiki.servarr.com/Radarr_Supported_{0}",
```

Comment:
Lets change this and V3 version back to `{0}#{1}` for the new wiki... That way if we have a section dedicated to that specific Indexer/Client it will go straight there instead of the overall section.